smos/src/Smos/Instance.hs

```diff
@@ -1,23 +1,31 @@
 module Smos.Instance where
 
 import Conduit
-import qualified Graphics.Vty as Vty (Config (..), defaultConfig, mkVty)
-import Graphics.Vty.Output.TerminfoBased (setWindowSize)
+import qualified Graphics.Vty.CrossPlatform as Vty (mkVty)
+import qualified Graphics.Vty as Vty (defaultConfig)
+-- import qualified Graphics.Vty.Platform.Unix.Settings  as Vty(settingInputFd,settingOutputFd,defaultSettings) --
+-- import Graphics.Vty.Platform.Unix.Output.TerminfoBased  (setWindowSize)
 import Smos
 import Smos.Terminal
 import Smos.Types
 
+{-
+
+
 withSmosInstance :: (MonadUnliftIO m) => SmosConfig -> Maybe StartingPath -> (TerminalHandle -> m a) -> m a
```

Comment:
I think this function is only used for testing.
We could split out a `smos-e2e` package to put it in, then `smos-web-server` can use that and the tests can be in there.